transport/control.go

```diff
@@ -56,43 +56,33 @@ type windowUpdate struct {
 	increment uint32
 }
 
-func (windowUpdate) isItem() bool {
-	return true
-}
+func (*windowUpdate) isItem() {}
```

Comment:
See the commit message https://github.com/tamird/grpc-go/commit/7e7145d85838627ba8531dc2f77623ef1a5b573a